great_expectations/profile/base.py

```diff
@@ -0,0 +1,55 @@
+import json
+import time
+
+from ..dataset import Dataset
+
+
+class DataSetProfiler(object):
+
+    @classmethod
+    def validate_dataset(cls, dataset):
+        return issubclass(type(dataset), Dataset)
+
+    @classmethod
+    def add_expectation_meta(cls, expectation):
+        if not "meta" in expectation:
+            expectation["meta"] = {}
+
+        expectation["meta"][str(cls.__name__)] = {
+            "confidence": "very low"
```

Comment:
eventually, I want to make this numeric. No change for this PR

Reply:
agreed